.github/workflows/tests.yaml

```diff
@@ -435,7 +435,7 @@ jobs:
         run: ./replication_importing_with_backup.sh
   replication-tunable-consistency:
     name: Replication tunable consistency
-    runs-on: ubuntu-latest-4-cores
+    runs-on: ubuntu-latest-8-cores
```

Comment:
Does this really help? I remember you said that the problem was still occurring in the 8 cores environment, didn't it?

Reply:
No, that was another test case.

I think this might still help because the error I saw was that weaviate took longer to start.